mobile_verifier/src/heartbeats.rs

```diff
@@ -136,11 +187,31 @@ impl HeartbeatReward {
     ) -> impl Stream<Item = Result<HeartbeatReward, sqlx::Error>> + 'a {
         sqlx::query_as::<_, HeartbeatKey>(
             r#"
-            SELECT hotspot_key, cbsd_id, cell_type
+            WITH coverage_objs AS (
```

Comment:
Is it possible for this query to pull a new coverage_object that is attached to a heartbeat that came after the rewardable period?